thread/src/components/Common/CommonContainer.js

```diff
@@ -6,7 +6,7 @@ const CommonContainer = (props) => (
 
   <Container
     {...props}
-    style={{backgroundColor:'#8bc34a', padding:20}}
+    style={{backgroundColor:'#66bb6a', padding:20}}
```

Comment:
remove padding please

Reply:
the padding is necessary so that the elements don't stick to the edge @jordanrios94

Reply:
then apply padding to content element, not container